envoy/ssl/context_config.h

```diff
@@ -18,6 +18,14 @@
 namespace Envoy {
 namespace Ssl {
 
+// CompliancePolicy is defined in BoringSSL to enforce NIST policy restrictions.
+enum class CompliancePolicy {
```

Comment:
I think it would be fine to use the protobuf enum here instead of redefining a parallel enum

Reply:
Done, makes sense.

Reply:
You forgot to remove the enum definition here